### PR TITLE
Alma to main fixup for footer and nav

### PR DESCRIPTION
**Pull request by JPrevost**
(also a missing css change... it's really deleting 2 lines but my auto-linter is hard to avoid)

https://mitlibraries.atlassian.net/browse/LM-412

### Secrets

- [ ] All new secrets have been added to Pantheon tiers
- [ ] Relevant secrets have been updated in Github Actions
- [ ] All new secrets documented in README

### Documentation

- [ ] Project documentation has been updated
- [x] No documentation changes are needed

### Accessibility

- [ ] ANDI or Wave has been run in accordance to
      [our guide](https://mitlibraries.github.io/guides/basics/a11y.html) and
      all issues introduced by these changes have been resolved or opened as new
      issues (link to those issues in the Pull Request details above)

### Stakeholder approval

- [x] Stakeholder approval has been confirmed
- [ ] Stakeholder approval is not needed

### Dependencies

NO dependencies are updated

## Code Reviewer

- [ ] The commit message is clear and follows our guidelines
      (not just this pull request message)
- [ ] The changes have been verified
- [ ] The documentation has been updated or is unnecessary
- [ ] New dependencies are appropriate or there were no changes

### CODE REVIEW

**Comment by matt-bernhardt** [comment]
Ooh, this could allow us to test LM-278 it seems. Let me check the multidev to confirm.

File: web/app/themes/mitlib-parent/css/scss/partials/_home.scss [hunk 1]
```diff
@@ -40,197 +50,242 @@ body.page-home {
...
+
 	//
 	// Hours & Locations
 	//
 	.hours-locations {
 		display: flex;
 		flex-direction: column;
+
 		h2 {
-			border-bottom: 1px solid $color-secondary;
```
Comment (matt-bernhardt) [inline web/app/themes/mitlib-parent/css/scss/partials/_home.scss:50]: This change will definitely allow us to confirm the workflow around deploying updates to the compiled stylesheets.

**Comment by JPrevost** [comment]
@matt-bernhardt I mostly can't figure out where that change even shows up, I just brought it forward as it looks like we missed that commit from legacy at some point... but you are correct that we can at least check the compiled css to see how low it takes to show up

File: web/app/themes/mitlib-parent/inc/footer-main.php [hunk 2]
```diff
@@ -1,23 +1,20 @@
 <?php
 /**
- * The template for the main footer, showing EDS / Barton as discovery options.
+ * The template for the main footer, showing Alma / Primo as discovery options.
  *
- * @package MITlib_Parent
- * @since 0.0.1
+ * @package MIT_Libraries_Parent
+ * @since 1.11.0
```
Comment (matt-bernhardt) [inline web/app/themes/mitlib-parent/inc/footer-main.php]: I'm open to hearing that this is too fastidious, but could we revert the changes to these file tags? The values here are applicable to the legacy theme, which we're breaking from now in a way that I was hoping would be consistent across the new theme.

File: web/app/themes/mitlib-parent/inc/footer-main.php [hunk 3]
```diff
@@ -1,23 +1,20 @@
...
  */
 
-namespace Mitlib\Parent;
```
Comment (matt-bernhardt) [inline web/app/themes/mitlib-parent/inc/footer-main.php]: Ditto for this - while I don't think anything in this template calls anything namespaced at the moment, I'd like to keep this line in place for consistency and in case we ever do need to call any PHP functions down the road.

File: web/app/themes/mitlib-parent/inc/footer-main.php [hunk 4]
```diff
@@ -1,23 +1,20 @@
...
 		<div class="flex-item">
 			<h4><a href="/search">Search</a></h4>
-			<a href="/quicksearch" class="link-sub">Quick search</a>
-			<a href="/barton" class="link-sub">Barton catalog</a>
-			<a href="/vera" class="link-sub">Vera: E-journals &amp; databases</a>
+			<a href="/search-collections" class="link-sub">Search Our Collections</a>
 			<a href="/worldcat" class="link-sub">WorldCat</a>
-			<a href="/barton-reserves" class="link-sub">Course reserves</a>
+			<a href="https://libguides.mit.edu/az.php" class="link-sub">Databases A-Z</a>
+			<a href="/search-reserves" class="link-sub">Course reserves</a>
```
Comment (matt-bernhardt) [inline web/app/themes/mitlib-parent/inc/footer-main.php:19]: For the rest of these markup changes, I'm trusting that UX has confirmed that this is what's needed.

File: web/app/themes/mitlib-parent/inc/nav-main.php [hunk 5]
```diff
@@ -2,8 +2,8 @@
 /**
  * Template for top navigation, showing Alma / Primo as discovery options.
  *
- * @package MITlib_Parent
- * @since 0.0.1
+ * @package MIT_Libraries_Parent
+ * @since 1.11.0
```
Comment (matt-bernhardt) [inline web/app/themes/mitlib-parent/inc/nav-main.php]: Same comment here as for `footer-main.php`.

**Review by matt-bernhardt — CHANGES_REQUESTED** [review]
The focus of this change is not concerning - and I'm glad you caught that I apparently used the wrong template when building out this theme.

The change I'd like to request is to preserve the file tags and namespace in the two partials, to keep things consistent with the rest of the new theme. I'm open to hearing that this is too fastidious of me, however.

I'm looking forward to deploying this and getting a good test case for LM-278, though. Thank you!

File: web/app/themes/mitlib-parent/inc/footer-main.php (hunk 3 above)
Comment (JPrevost) [reply web/app/themes/mitlib-parent/inc/footer-main.php]: @matt-bernhardt I un-removed the namespace for this file as you requested but also added it to the other file where it wasn't before. I'm fairly certain that's probably fine but also if not let me know!

File: web/app/themes/mitlib-parent/inc/footer-main.php (hunk 4 above)
Comment (JPrevost) [reply web/app/themes/mitlib-parent/inc/footer-main.php:19]: yup

**Review by matt-bernhardt — APPROVED** [review]
Yep - this looks good. Good catch on the namespace on the other file, I didn't notice that it wasn't present. Consistency FTW!

:shipit:
